README.md

```diff
@@ -10,6 +10,8 @@ Example usage of this can be found here
  * [gst-plugin-rs](https://github.com/sdroege/gst-plugin-rs): GStreamer plugin
    writing infrastructure
  * [gtk-subclass](https://github.com/sdroege/gtk-subclass): GTK class bindings
+ * [Listbox Model Data implementation](https://github.com/gtk-rs/examples/raw/master/src/bin/listbox_model.rs):
+   This is a good, quick example of how to use this crate for gtk-rs related implementations.
```

Comment:
Maybe want to say that this is a example of "how to use this crate for making use of a `gio::ListStore` as a model of a `gtk::ListBox`" or something like that. The gobject-subclass usage in there is completely unrelated to GTK :)

Reply:
:+1: